config/routes.rb

```diff
@@ -15,4 +15,12 @@
       end
     end
   end
-end
\ No newline at end of file
+
+  namespace :api do
+    post '/paypal', :to => "paypal#express", :as => :paypal_express_api
+    post '/paypal/confirm', :to => "paypal#confirm", :as => :confirm_paypal_api
+    post '/paypal/cancel', :to => "paypal#cancel", :as => :cancel_paypal_api
+    get '/paypal/notify', :to => "paypal#notify", :as => :notify_paypal_api
```

Comment:
there isnt one I just copied that from the the other paypal stuff I think it was some old thing that never got removed. I would search the history of the repository for it